### ISSUE TITLE
docs.flutter.io mobile issue 

### ISSUE THREAD

**Issue by jcollins-g** [dart-lang/dartdoc#1911]
_From @feinstein on January 25, 2019 15:50_

Page URL: https://docs.flutter.io/flutter/widgets/LocalizationsDelegate-class.html

Description:

The docs are not rendering perfectly on a mobile device. 

I am not sure if this is the right repo to report it actually 

_Copied from original issue: flutter/website#2274_

**Comment by jcollins-g** [comment]
_From @chalin on January 25, 2019 15:59_

Here's a screenshot:
<img width="503" alt="screen shot 2019-01-25 at 10 57 54" src="https://user-images.githubusercontent.com/4140793/51756971-3b132280-2090-11e9-983d-d9fd840acab0.png">
@jcollins-g @filiph @Hixie /cc @sfshaza2 @kwalrath

**Comment by jcollins-g** [comment]
Have you tried https://master-docs-flutter-io.firebaseapp.com/ ?   I fixed several critical mobile-blocking bugs since the last stable release, hopefully the next release will improve matters.

**Comment by jcollins-g** [comment]
dartdoc omnibus mobile fix PR: https://github.com/dart-lang/dartdoc/pull/1873
Push to Flutter:  https://github.com/flutter/flutter/pull/26041

**Comment by jcollins-g** [comment]
_From @feinstein on January 25, 2019 17:21_

https://master-docs-flutter-io.firebaseapp.com/ has the same error

**Comment by jcollins-g** [comment]
I was able to reproduce this in simulator and with a real device.

This seems to be some artifact of how dartdoc is doing scrolling.  If you attempt to swipe the entire page to the left in a particular way, the doc page can get stuck there.  Worse, the stickiness seems to be persistent and reloads of the page don't seem to help.

A workaround is to manually swipe the page back to the right, but that only works on https://master-docs-flutter-io.firebaseapp.com/.  docs.flutter.io is stuck for good when this happens.

**Comment by feinstein** [comment]
I also found the same issue at the Dart Pub https://pub.dartlang.org/packages/flutter_google_places_autocomplete

**Comment by jcollins-g** [comment]
Yes.  I believe all dartdoc generated documentation will do this on mobile.

**Comment by jcollins-g** [comment]
closing with #1916
